src/go/rpk/pkg/cli/cluster/storage/recovery/recovery.go

```diff
@@ -17,19 +17,20 @@ import (
 
 func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "recovery",
-		Short: "Interact with the topic recovery process",
-		Long: `Interact with the topic recovery process.
+		Use:     "restore",
```

Comment:
Wonder if we should use `restoration` instead of restore, usually rpk goes: `rpk <noun> <verb>` but with restore we are breaking this rule.

Reply:
I'm personally more a fan of "restore" over "restoration". In my head the noun version of running this sounds like "start the restore process" rather than "start the restoration"